examples/get-started/pure-js/google-maps/app2.js

```diff
@@ -0,0 +1,74 @@
+/* global document, google, window */
```

Comment:
The get-started examples are intended to be boilerplates, not showcases of library capabilities. Let's keep the dataset aligned with all other get-started examples. For dev testing purpose, you can add apps to test/apps. We will make a full standalone demo of context interleaving by updating https://deck.gl/examples/google-maps/

Reply:
Agree. I think we can remove the examples in this PR. Once we land this, we can work on a cool 3D example to replace https://deck.gl/examples/google-maps

Reply:
I've removed the examples from the PR